src/language/from_relay/run_relay.py

```diff
@@ -31,7 +30,20 @@
     relay_in = parsed.relay_filepath.read()
     expr = tvm.parser.fromtext(relay_in)
 
-inputs = [np.load(arg_file) for arg_file in parsed.npy_arg_filepath]
-output = relay.create_executor(mod=expr).evaluate()(*inputs).asnumpy()
+inputs = []
+for filepath in parsed.npy_arg_filepath:
+    with open(filepath, 'rb') as arg_file:
+        inputs.append(np.load(arg_file))
 
-np.save(parsed.npy_out_filepath, output)
+# need graph runtime or crashes for yolo
+output = relay.create_executor(mod=expr, kind="graph").evaluate()(*inputs)
+
+if type(output) is list:
```

Comment:
Can you add an assert here, checking that the length of the Relay output is the same as the number of output filepaths given to the program?